wcivf/apps/people/templates/people/includes/_person_intro_card.html

```diff
@@ -1,42 +1,48 @@
-{%  load humanize %}
+{% load humanize %}
+{% load i18n %}
+
 <section class="ds-candidate">
     <div class="ds-candidate-body ds-stack-smaller">
         <h2 class="ds-candidate-name ds-h3">
             {{ object.name }}
         </h2>
         {% if person.display_deceased and person.featured_candidacy.party.is_independent %}
-            (Deceased)
+            {% trans "(Deceased)" %}
         {% endif %}
 
         {% if object.current_or_future_candidacies.all.count > 1 %}
-            <h5>{{ object.name }} is {% if object.current_or_future_candidacies.0.party.is_independent %}an{% else %}a{% endif %}
-                {{object.current_or_future_candidacies.0.party_name }} candidate in the following elections:</h5>
+            <h5>{% blocktrans with party=object.current_or_future_candidacies.0.party_name %}{{ object.name }} is {% if object.current_or_future_candidacies.0.party.is_independent %}an{% else %}a{% endif %}
+                {{ party}} candidate in the following elections:{% endblocktrans %}</h5>
 
             {% for candidacy in object.current_or_future_candidacies %}
                 <ul>
-                    <li>
-                        {{ candidacy.election.name }} for <a href="{{ candidacy.post_election.get_absolute_url }}">
-                            {{ candidacy.post_election.friendly_name }}</a>
-                        {% if candidacy.votes_cast %}
-                            {{ candidacy.votes_cast|intcomma }} votes
-                        {% endif %}
-                        {% if candidacy.elected %}
-                            (elected)
-                        {% endif %}
-                    </li>
-                </ul>
-            {% endfor %}
+                    <li>{% blocktrans with
+                        election=candidacy.election.name
+                        election_name=candidacy.post_election.friendly_name
+                        num_votes=candidacy.votes_cast|intcomma
+                        %}{{ election }} for</li>
+                    <a href="{{ candidacy.post_election.get_absolute_url }}">
+                        {{ election_name }}</a>
+                    {% if candidacy.votes_cast %}
+                        {{ num_votes }} votes
+                    {% endif %}
+                    {% if candidacy.elected %}
+                        (elected)
+                    {% endif %}
+            {% endblocktrans %}</li>
+            </ul>
+        {% endfor %}
 
-        {% else %}
-            <p>
-                {{ object.intro|safe }}
-            </p>
-        {% endif %}
-    </div>
+{% else %}
+    <p>
+        {% blocktrans %}{{ object.intro|safe }}{% endblocktrans %}
```

Comment:
`intro` is created in the model. Should the translation be done there instead?

Reply:
Yes, I think so